channels/rdpecam/client/camera.h

```diff
@@ -105,6 +111,11 @@ typedef struct
 	AVFrame* avOutFrame;
 #endif
 
+#if defined(WITH_INPUT_FORMAT_H264)
+	size_t h264FrameMaxSize;
```

Comment:
missing `#include <stddef.h>` or `winpr/wtypes.h` for the `size_t` and other types